Gruntfile.js

```diff
@@ -310,10 +310,6 @@ module.exports = function(grunt) {
       js: {
         files: ['Gruntfile.js', '<%= loc.src %>/static/js/**/*.js'],
         tasks: ['js']
-      },
-      content: {
-        files: ['**/*.html', '/downloads/*.md', '/guides/*.md', '/identity/*.md', '/ui-toolkit/*.md'],
-        tasks: ['copy']
```

Comment:
Yeah, look carefully. The `copy:main` subtask is the only subtask of copy. The individual steps in it cannot be run individually currently. The first of these parts, the "HTML files" one, could be removed, though.